src/main/java/com/codeborne/selenide/webdriver/AbstractDriverFactory.java

```diff
@@ -74,7 +75,7 @@ protected <T extends MutableCapabilities> T createCommonCapabilities(T capabilit
     if (browser.supportsInsecureCerts()) {
       capabilities.setCapability(ACCEPT_INSECURE_CERTS, true);
     }
-    capabilities.setCapability(UNHANDLED_PROMPT_BEHAVIOUR, "accept");
+    capabilities.setCapability(UNHANDLED_PROMPT_BEHAVIOUR, ACCEPT_AND_NOTIFY);
```

Comment:
I am opposed to changing the UNHANDLED_PROMPT_BEHAVIOUR, as it has been the default for many years and no one has blamed it for any issues. Changing it would break the tests of many users, and they do not want to have to handle the repercussions of such a change.

Instead, we have to make sure that selenide don't hide UnhandledAlertException. It will cover all user cases.

Reply:
That's the problem. If we leave "accept", then webdriver doesn't throw `UnhandledAlertException`. 

For users who don't want to investigate, there is still an option to to set `capabilities.setCapability(UNHANDLED_PROMPT_BEHAVIOUR, ACCEPT);` in their tests.

Reply:
>That's the problem. If we leave "accept", then webdriver doesn't throw `UnhandledAlertException`.

Why problem? It is awesome! No one cares about shitty alerts for years and now you break tests with version update, users spend time how to set UNHANDLED_PROMPT_BEHAVIOUR back. For what? If user tests allerts there is api for that.

If someone wants `UnhandledAlertException` there is still an option to to set `capabilities.setCapability(UNHANDLED_PROMPT_BEHAVIOUR, ACCEPT_AND_NOTIFY);`

Reply:
Agree, we can leave `"accept"` by default, and users will be able to set `ACCEPT_AND_NOTIFY` if they wish.

P.S. Why you call alerts "shitty"? They might show some real errors that should not be ignored. If we ignore these alerts, the SUT will  still behave incorrectly, but we would not even know about it.

Reply:
>They might show some real errors that should not be ignored. If we ignore these alerts, the SUT will still behave incorrectly, but we would not even know about it.

That's true. But alerts can sometimes be perceived as spam or useless if they are not relevant or timely for the user. But developers don't care and use them(e.g. in legacy CRM\CMS). In such a case it is ok to ignore it in the test.

Also incorporating excessive alerts into the user interface is a poor design decision from a usability and user experience standpoint, as it is often difficult to provide helpful information to the user through error messages displayed in alert form. For example, it is not possible to include clickable URLs in an alert, which can make it harder for the user to access additional resources or information that may be necessary to resolve an issue.
That's why alerts "shitty" :)